packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/navigation-menu-content.js

```diff
@@ -8,24 +13,59 @@ import {
 	BlockTools,
 } from '@wordpress/block-editor';
 import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
 
 /**
  * Internal dependencies
  */
 import { unlock } from '../../private-apis';
 import { NavigationMenuLoader } from './loader';
 
+// Needs to be kept in sync with the query used at packages/block-library/src/page-list/edit.js.
```

Comment:
Should we add a similar comment on the other file too, to avoid people missing this if they change the other file. Also, maybe add a sentence here to explain why we use it. Some thing like

"We use the query to only render the menus once everything is loaded." (or something like that)